config/settings/base.py

```diff
@@ -0,0 +1,91 @@
+"""
+Django settings for mfl_api project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.6/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.6/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.6/howto/deployment/checklist/
```

Comment:
Many of the "helpful comments" in the settings file can be removed;

Also - was the `startproject` run with Django 1.7 or 1.8?